src/pages/404/404.ts

```diff
@@ -0,0 +1,18 @@
+import {Block} from "../../shared/utils";
+import tmpl from './404.tmpl.hbs'
+import '../../scss/styles.scss'
+import '../../scss/pages/service-page.scss'
+export class NotFound extends Block {
```

Comment:
```suggestion
import '../../scss/pages/service-page.scss'

export class NotFound extends Block {
```

Обычно между импортами и началом кода, добавляют пустую строку